documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
+The identifier contains all information to resolve a DID, providing the latest DID Document. The first three characters `did` indicate that the DID standard from W3C must be used to resolve the identifier. It is followed by a unique method name, in our case `iota`, to indicate that the IOTA method is used. The IOTA method is a specific implementation that follows the following [method spec](../specs/did/iota_did_method_spec.md). This provides unique rules for the protocol to follow to result in the latest DID Document. In our case, it describes how DID Documents are uploaded and queried to and from the IOTA Tangle. Lastly, a DID contains a set of random characters that are unique per identity, making the identity unique and ensuring every identity resolves to a unique DID Document. 
 
-:::tip Requires basic knowledge of Asymmetric Encryption
+:::note Requires basic knowledge of Asymmetric Encryption
 
 The following and later sections require some basic knowledge of Asymmetric Encryption. Please read or view some materials on the subject before continuing.
 
 :::
 
-A DID Document mostly contains two important pieces of data: public keys and services. The public keys can be used to prove ownership over the identity, by cryptographically signing something with the associated private key. The public key can be used to verify that the identity subject signed the data and therefore controls the private key. Ownership over the private keys, therefore, proves ownership over the identity. This also means that it is very important to keep the private keys safe and secure. In addition, the public keys allow users to send encrypted data to the identity, using their public key, that only the identity owner can decrypt.
+A DID Document contains at least two important pieces of data: public keys and services. The public keys can be used to prove ownership over the identity, by cryptographically signing something with the associated private key. The public key can be used to verify that the identity subject signed the data and therefore controls the private key. Ownership over the private keys, therefore, proves ownership over the identity. This also means that it is very important to keep the private keys safe and secure. Additionally, the public keys allow users to send encrypted data to the identity, using their public key, that only the identity owner can decrypt.
```

Comment:
From here and on: I don't think that changing a phrase to an adverb is a good change in general.

Reply:
Hm. @lucas-tortora , tie breaker.

Reply:
I'd go with adverbs, but I don't think it makes much of a difference one way or the other.